s3tests_boto3/functional/test_s3select.py

```diff
@@ -242,6 +248,127 @@ def test_column_sum_min_max():
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select int(_1) from stdin where _1 is null;')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select int(_2) from stdin where _1 == "null";')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select _1 from stdin where _1 like "a%";')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select _1 from stdin where substr(_1,1,1) == "a";')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select _1 from stdin where _1 like "ab%";')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select _1 from stdin where substr(_1,1,1) == "a" and substr(_1,2,1) == "b";')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select _1 from stdin where _1 like "[d-f]bcde";')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select _1 from stdin where substr(_1,1,1) == "d" or substr(_1,1,1) == "e" or substr(_1,1,1) == "f" and substr(_1,2,1) == "b" and substr(_1,3,1) == "c" and substr(_1,4,1) == "d" and substr(1,5,1) == "e";')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select _2 from stdin where _2 like "%ab";')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select _2 from stdin where substr(_2,-1,1) == "b" and substr(_2,-2,1) == "a";')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select _3 from stdin where _3 like "a%b";')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select _3 from stdin where substr(_3,-1,1) == "b" and substr(_3,1,1) == "a";')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
+
+    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select _2 from stdin where _2 like "ab_";')).replace("\n","")
+
+    res_s3select = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name, 'select _2 from stdin where substr(_2,1,1) == "a" and substr(_2,2,1) == "b" and len(_2)==3;')).replace("\n","")
+
+    nose.tools.assert_equal( res_s3select_in, res_s3select )
```

Comment:
it's not clear what is the input 
in the case the input contain only numeric characters, the result is zero rows for both queries.